### ISSUE TITLE
moov atom not found 160906_band3 03 view

### ISSUE THREAD

**Issue by twangnh** [CMU-Perceptual-Computing-Lab/panoptic-toolbox#49]
Hi I tried download the dataset, however, the view-03 video of sequence 160906_band3 seems broken, when trying to extract  hdimages, it raises **moov atom not found error**

**Comment by geniuspatrick** [comment]
Have you solved the problem? I had the same problem. This file appears to be corrupted.

**Comment by twangnh** [comment]
yes, I have contacted the maintainers of the dataset, they also cannot fix this problem, the video recording is broken.

**Comment by cucdengjunli** [comment]
Have you solved the problem? I had the same problem. This file appears to be corrupted.

**Comment by Kairobo** [comment]
Please help me if you solved it. #66
